src/__test__/streams/appendToStream-errors.test.ts

```diff
@@ -34,7 +34,11 @@ describe("appendToStream - errors", () => {
   });
 
   describe.each([
-    ["Batch append (>21.10)", matchServerVersion`>=21.10`, undefined],
+    [
+      "Batch append (>21.10)",
+      matchServerVersion`>=21.10`,
+      { username: "admin", password: "changeit" },
```

Comment:
This test no longer makes sense, as it was using the inline credentials to force a "normal" append, vs a batch append. 

So, the credentials should be added in client creation.